src/test/setupUtil.ts

```diff
@@ -56,8 +57,14 @@ export function setRunnableTimeout(test: Mocha.Runnable, maxTestDuration: number
 }
 
 export function skipTest(testOrCtx: Mocha.Context | Mocha.Test | undefined, reason?: string) {
-    const test =
-        testOrCtx?.type === 'test' ? (testOrCtx as Mocha.Test) : (testOrCtx as Mocha.Context | undefined)?.currentTest
+    let test
+
+    if (testOrCtx?.type === 'test') {
+        test = testOrCtx as Mocha.Test
+    } else {
+        const context = testOrCtx as Mocha.Context | undefined
+        test = context?.currentTest ?? context?.test
```

Comment:
The ternary was nice because it allows `const`, and type inference. But it's probably too messy to add a nested ternary.